src/components/FlexibleLink.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <a v-if="isExternal" 
+    v-bind="{...$props, ...$attrs}"
+    :href="to" 
+    target="_blank" 
```

Comment:
we should add a prop "target" that is "_blank" by default

Reply:
I think external links will always be target blank, so that's probably not necessary - right?

Reply:
Would be nice to have that flexibility.

Reply:
I think we might want the possibility eventually but we can build that out when we need it.